sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java

```diff
@@ -41,10 +46,146 @@ public CreateNewEventSteps(WebDriverHelpers webDriverHelpers) {
           webDriverHelpers.fillInWebElement(
               TITLE_INPUT, "EVENT_AUTOMATION" + timestamp + Faker.instance().name().name());
           webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
-          webDriverHelpers.waitUntilElementIsVisibleAndClickable(ADD_PARTICIPANT_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NEW_EVENT_CREATED_MESSAGE);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+        });
+    When(
+        "^I create a new event with specific data$",
+        () -> {
+          newEvent = eventService.buildGeneratedEvent();
+          fillDateOfReport(newEvent.getReportDate());
+          fillStartData(newEvent.getEventDate());
+          selectEventStatus(newEvent.getEventStatus());
+          selectEventInvestigationStatusOptions(newEvent.getInvestigationStatus());
+          selectEventInvestigationStatusOptions(
+              newEvent.getInvestigationStatus()); // remove after bug 5547 is fixed the duplication
+          selectEventManagementStatusOption(newEvent.getEventManagementStatus());
+          selectRiskLevel(newEvent.getRiskLevel());
+          selectDisease(newEvent.getDisease());
+          fillTitle(newEvent.getTitle());
+          selectSourceType(newEvent.getSourceType());
+          selectTypeOfPlace(newEvent.getEventLocation());
+          newEvent =
+              newEvent.toBuilder()
+                  .uuid(webDriverHelpers.getValueFromWebElement(UUID_INPUT))
+                  .build();
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NEW_EVENT_CREATED_MESSAGE);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+        });
+    When(
+        "^I create a new event with status ([^\"]*)",
+        (String eventStatus) -> {
+          newEvent = collectEventUuid();
+          String timestamp = String.valueOf(System.currentTimeMillis());
+          webDriverHelpers.fillInWebElement(TITLE_INPUT, "EVENT_AUTOMATION" + timestamp);
+          selectEventStatus(eventStatus);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(NEW_EVENT_CREATED_MESSAGE);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+        });
+    When(
+        "^I validate create a new event popup",
```

Comment:
after naming, I don't understand if we validate elements or update fields

Reply:
changed to: I validate all fields are rendered correctly on create a new event popup window